Create Blog Post “q2-2023-releases” (#2477)
Nexmo/deved-platform: content/blog/en/q2-2023-releases.md

outdated: false
replacement_url: ""
---
And just like that, we're halfway into 2023! Our Vonage DevRel team wrapped up another season of dropping game-changing releases. Let's check them out.

## [Live Captions API Is in Beta!](https://developer.vonage.com/en/blog/live-captions-api-is-in-beta)

[Dwane](https://developer.vonage.com/en/blog/authors/dwanehemmings) announces the Beta release of Vonage's Live Captions API, now ready to run in your applications. Live captioning provide many benefits. They improve accessibility for users by providing dialogue visually when environments are noisy. They allow developers the ability to provide real-time multilingual translation.

## [Vonage Video API - Secure Callbacks Public Beta Announcement](https://developer.vonage.com/en/blog/vonage-video-api-secure-callbacks-public-beta-announcement)

Our Video API team has rolled out a new feature called Secure Callbacks, currently in public beta and anticipated for a full launch. Michael explains how the feature validates webhook callback requests from Vonage and ensures their payload hasn't been tampered with. It's a big win for app security and also a plus for compliance-heavy industries.

> ## [Vonage Video API's Secure Callbacks Now in Public Beta](https://developer.vonage.com/en/blog/vonage-video-api-secure-callbacks-public-beta-announcement)

Reviewer: ```suggestion
```

> The React Native SDK for the Vonage Video API transitions from community to official Vonage support! This means improved, consistent assistance for developers and boosted confidence when integrating the SDK, assured of routine updates and long-term maintenance. Need help? Our Vonage support team is ready to assist. For guides and samples, check our official documentation, the React Native package on npm, and the SDK source code on the SDKs GitHub repository.

Reviewer: ```suggestion
```

## [Vonage Video API - React Native SDK Gains Official Support](https://developer.vonage.com/en/blog/vonage-video-api-react-native-sdk-gains-official-support)

> 

Reviewer: ```suggestion
The React Native SDK for the Vonage Video API transitions from community to official Vonage support! This means improved, consistent assistance for developers and boosted confidence when integrating the SDK, assured of routine updates and long-term maintenance. Need help? Our Vonage support team is ready to assist. For guides and samples, check our official documentation, the React Native package on npm, and the SDK source code on the SDKs GitHub repository.

```

After an awesome Beta ride, the Vonage Video API MacOS SDK is now officially making its mark with its first-ever stable release. Now, you can flexibly use the MacOS SDK in production, and expect the same maintenance and frequent updates as our other SDKs (JavaScript, Android, iOS, Windows, and Linux). Whether you're just starting out or eager to dive straight into the code, we've got you covered. Our official documentation and MacOS SDK sample repo are filled with all the details and sample apps to kick-start your journey with the SDK.

## [New Vonage Client SDKs for Android and iOS](https://developer.vonage.com/en/blog/introducing-the-new-vonage-client-sdk-for-android-and-ios)

We're revving things up in Q2 with the grand unveiling of the newly renamed Vonage Client SDK! It's coming in hot with a new major version full of fresh features and fixes. This SDK now uses Kotlin Multiplatform, sharing core functionality between Android and iOS SDKs.

Our developer advocate Abdul covers key concepts, including Session, Invite, Call, and Leg, to familiarize you with its workings. He also guides you through creating a session, making and receiving a call, and push notifications.

> 

Reviewer: ```suggestion

## [Announcing Vonage Proactive Connect GA](LINK)

Prepare to level-up your outreach campaigns with Proactive Connect! Seamlessly integrating with multiple Vonage Communications APIs - including SMS, Messages, Voice, and AI Studio - Proactive Connect lets you design large-scale, tailor-made campaigns across diverse channels. This API not only enables message scheduling and personalization, but also allows for effective customer response management within your Vonage Dashboard.

```

> ## [Vonage Verify V2 Is Now GA for 2FA Integrations](https://developer.vonage.com/en/blog/vonage-verify-v2-is-now-ga-for-2fa-integrations)

Reviewer: ```suggestion

## [Cloud Runtime Marketplace in Beta](https://developer.vonage.com/en/blog/announcing-cloud-runtime-marketplace)
We're thrilled to introduce the Beta availability of the Vonage [Cloud Runtime Marketplace](https://developer.vonage.com/cloud-runtime), our innovative cloud-native, serverless development platform! Vonage Cloud Runtime, the very same tool that our Solutions and Services teams use to deploy API interactions, is now at your fingertips. We have easy-to-use guides and tools that take your application build, deployment, and production from zero to hero. And here's the kicker - we host this serverless platform, so you don't have to worry about maintaining infrastructure to use Vonage APIs! During this Beta period, there's no fee to access the Vonage Cloud Runtime, so you can dive right in and start crafting extraordinary customer engagement applications today!

## [Vonage Verify V2 Is Now GA for 2FA Integrations](https://developer.vonage.com/en/blog/vonage-verify-v2-is-now-ga-for-2fa-integrations)

```

---
title: Q2 2023 Releases
description: All Vonage releases from April to June 2023
author: diana-pham-1
published: true
published_at: 2023-06-12T21:35:45.175Z
updated_at: 2023-06-12T21:35:45.189Z
category: community
tags:
  - releases
  - video-api
  - verify-api
comments: true
spotlight: false
redirect: ""
canonical: ""
outdated: false
replacement_url: ""
---
And just like that, we're halfway into 2023! Our Vonage DevRel team wrapped up another season of dropping game-changing releases. Let's check them out.

## [Live Captions API Is in Beta!](https://developer.vonage.com/en/blog/live-captions-api-is-in-beta)

> Our very own JS developer advocate, Dwane, announces the Beta release of Vonage's Live Captions API, now ready to run in your applications. Dwane illustrates the essential benefits of live captions: their crucial role in fostering accessibility, counteracting the nuisance of noisy environments, forming a foundation for translation, and the ability to recapture any lost dialogue in a conversation. Notably, he cites a YouGov poll that unveils that a good 28% of people appreciate having subtitles on.

Reviewer: ```suggestion
[Dwane](https://developer.vonage.com/en/blog/authors/dwanehemmings) announces the Beta release of Vonage's Live Captions API, now ready to run in your applications. Live captioning provide many benefits. They improve accessibility for users by providing dialogue visually when environments are noisy. They allow developers the ability to provide real-time multilingual translation.
```

## [Vonage Video API - Secure Callbacks Public Beta Announcement](https://developer.vonage.com/en/blog/vonage-video-api-secure-callbacks-public-beta-announcement)

Our Video API team has rolled out a new feature called Secure Callbacks, currently in public beta and anticipated for a full launch. Michael explains how the feature validates webhook callback requests from Vonage and ensures their payload hasn't been tampered with. It's a big win for app security and also a plus for compliance-heavy industries.

> The Vonage Video API uses callbacks for Session Monitoring, Archiving/Recording, SIP, and Experience Composer. These callbacks are now signed by a user-configured signature, confirming they come from Vonage.

Reviewer 2: Would remove. I think the previous paragraph is enough. This feels like an uncessary dangling bit.

> The Vonage Video API uses callbacks for Session Monitoring, Archiving/Recording, SIP, and Experience Composer. These callbacks are now signed by a user-configured signature, confirming they come from Vonage.

Reviewer: ```suggestion
```

> ## [Vonage Video API's Secure Callbacks Now in Public Beta](https://developer.vonage.com/en/blog/secure-callbacks-beta)

Reviewer: ```suggestion
## [Vonage Video API's Secure Callbacks Now in Public Beta](https://developer.vonage.com/en/blog/vonage-video-api-secure-callbacks-public-beta-announcement)
```

> In an exciting update by our Android dev advocate Zachary, the React Native SDK for the Vonage Video API transitions from community to official Vonage support! This means improved, consistent assistance for developers and boosted confidence when integrating the SDK, assured of routine updates and long-term maintenance. Need help? Our Vonage support team is ready to assist. For guides and samples, check our official documentation, the React Native package on npm, and the SDK source code on the SDKs GitHub repository.

Reviewer 2: Again, don't think it's necessary to mention Zach here.

Reviewer 2: Wrong article in the section header.

> In an exciting update by our Android dev advocate Zachary, the React Native SDK for the Vonage Video API transitions from community to official Vonage support! This means improved, consistent assistance for developers and boosted confidence when integrating the SDK, assured of routine updates and long-term maintenance. Need help? Our Vonage support team is ready to assist. For guides and samples, check our official documentation, the React Native package on npm, and the SDK source code on the SDKs GitHub repository.

Reviewer: ```suggestion
The React Native SDK for the Vonage Video API transitions from community to official Vonage support! This means improved, consistent assistance for developers and boosted confidence when integrating the SDK, assured of routine updates and long-term maintenance. Need help? Our Vonage support team is ready to assist. For guides and samples, check our official documentation, the React Native package on npm, and the SDK source code on the SDKs GitHub repository.
```

## [Vonage Video API - React Native SDK Gains Official Support](https://developer.vonage.com/en/blog/vonage-video-api-react-native-sdk-gains-official-support)

> Exciting times ahead as Zachary reveals that Vonage Video API's React Native SDK has graduated from community support to being officially backed by Vonage! This shift is a game-changer, offering developers dependable, top-tier assistance and increased reliability when implementing the SDK, all thanks to committed updates, maintenance plans, and our team's support. To get started or for sample references, check out our official documentation, explore the React Native package on npm, or sift through the SDK source code available on the GitHub repository.

Reviewer 2: Seems you've duplicated the previous section but in a different version? I prefer the first version above. But please fix the section header.

> Exciting times ahead as Zachary reveals that Vonage Video API's React Native SDK has graduated from community support to being officially backed by Vonage! This shift is a game-changer, offering developers dependable, top-tier assistance and increased reliability when implementing the SDK, all thanks to committed updates, maintenance plans, and our team's support. To get started or for sample references, check out our official documentation, explore the React Native package on npm, or sift through the SDK source code available on the GitHub repository.

Reviewer: ```suggestion
```

> ## [Vonage Video API MacOS SDK Hits General Availability](https://developer.vonage.com/en/blog/vonage-video-api-macos-sdk-goes-ga)

Reviewer 2: MacOS should be macOS. I've let @DanyF-github know the original article is wrong.

> ## [Vonage Video API MacOS SDK Hits General Availability](https://developer.vonage.com/en/blog/vonage-video-api-macos-sdk-goes-ga)

Reviewer: ```suggestion
## [Vonage Video API macOS SDK Hits General Availability](https://developer.vonage.com/en/blog/vonage-video-api-macos-sdk-goes-ga)
```

---

> title: Q2 2023 Releases

Reviewer: Change title

outdated: false
replacement_url: ""
---
And just like that, we're halfway into 2023! Our Vonage DevRel team wrapped up another season of dropping game-changing releases. Let's check them out.

## [Live Captions API Is in Beta!](https://developer.vonage.com/en/blog/live-captions-api-is-in-beta)

> Our very own JS developer advocate, Dwane, announces the Beta release of Vonage's Live Captions API, now ready to run in your applications. Dwane illustrates the essential benefits of live captions: their crucial role in fostering accessibility, counteracting the nuisance of noisy environments, forming a foundation for translation, and the ability to recapture any lost dialogue in a conversation. Notably, he cites a YouGov poll that unveils that a good 28% of people appreciate having subtitles on.

Reviewer 2: Not sure if this sentence makes sense to me: Dwane illustrates the essential benefits of live captions: their crucial role in fostering accessibility, counteracting the nuisance of noisy environments, forming a foundation for translation, and the ability to recapture any lost dialogue in a conversation.

What does it mean to foster accessibility? Or form a foundation for translation? I would keep it simple. 

Live captioning provide many benefits. They improve accessibility for users by providing dialogue visually when environments are noisy. They allow developers the ability to provide real-time multilingual translation.
